bin/oneoff/nces_data/export_schools

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
```

Comment:
Is this a one-off script, or one we expect to use again in the future? 
If the latter, please move it up a directory and (perhaps in either case) add a little description at the top.

Reply:
I wanted to keep it with the other nces data scripts since you typically would export after you load data. But, it's still a one-off script.